cognite/seismic/protos/types.proto

```diff
@@ -35,6 +35,16 @@ message Trace {
     Coordinate coordinate = 5;
 }
 
+/**
+    Wrapper over Trace type that additionally provide information about the range of z values
+    included in the trace.
+ **/
+ message SlabTrace {
```

Comment:
Should we rather expand the Trace type?

Reply:
This information is already in any other trace - ie <0, trace.len()>. The trace in slab is a special case, since it's trimmed